portal-api/src/constants/date-formats.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * For use with date-fns format method.
+ *
+ * e.g. 1st February 2024
+ *
+ * Equivalent to 'Do MMMM YYYY' in moment().format()
+ *
+ * https://date-fns.org/v3.3.1/docs/format
+ */
+export const LONG_FORM_DATE = 'do MMMM yyyy';
```

Comment:
`do`? `dd`?

Reply:
(if this is `dd`, I'm a little surprised the tests are passing)

Reply:
`do` is for ordinal numbers (e.g. 1st, 2nd)